packages/@react-spectrum/picker/test/Picker.test.js

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {act, cleanup, fireEvent, render, within} from '@testing-library/react';
+import {Item, Picker} from '../src';
+import {Provider} from '@react-spectrum/provider';
+import React from 'react';
+import scaleMedium from '@adobe/spectrum-css-temp/vars/spectrum-medium-unique.css';
+import themeLight from '@adobe/spectrum-css-temp/vars/spectrum-light-unique.css';
+import {triggerPress} from '@react-spectrum/test-utils';
+
+let theme = {
+  light: themeLight,
+  medium: scaleMedium
+};
+
+describe('Picker', function () {
+  let offsetWidth, offsetHeight;
+  let onSelectionChange = jest.fn();
+
+  beforeAll(function () {
+    offsetWidth = jest.spyOn(window.HTMLElement.prototype, 'offsetWidth', 'get').mockImplementation(() => 1000);
+    offsetHeight = jest.spyOn(window.HTMLElement.prototype, 'offsetHeight', 'get').mockImplementation(() => 1000);
+    window.HTMLElement.prototype.scrollIntoView = jest.fn();
+    jest.spyOn(window, 'requestAnimationFrame').mockImplementation(cb => setTimeout(cb, 0));
+    jest.useFakeTimers();
+  });
+
+  afterEach(() => {
+    cleanup();
+  });
+
+  afterAll(function () {
+    offsetWidth.mockReset();
+    offsetHeight.mockReset();
+  });
+
+  it('can be opened', function () {
+    let {getAllByText, getAllByRole, getByRole} = render(
+      <Provider theme={theme}>
+        <Picker label="Test" onSelectionChange={onSelectionChange}>
+          <Item>One</Item>
+          <Item>Two</Item>
+          <Item>Three</Item>
+        </Picker>
+      </Provider>
+    );
+
+    let picker = getByRole('button');
+    act(() => triggerPress(picker));
```

Comment:
Why the `act`?

Reply:
Indirectly causes state changes, so I put it in an act, otherwise rtl gets angry and sends red messages to the console

Reply:
Hmm strange. Why haven't we come across this before? We fire lots of events in our tests...

Reply:
We have come across it, that's why our console is so full of red angry warning messages